spec/views/produce_items/show.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'produce_items/show', type: :view do
+  it 'display produce item' do
+    assign(:produce_item, ProduceItem.create!(name: 'banana', category: 'fruit'))
+    render
+    expect(rendered).to match(/banana/)
+  end
+
+  it 'displays months' do
+    @produce_item = ProduceItem.create!(name: 'banana', category: 'fruit')
+    assign(:item_seasons, [
+      ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 1, country_code: 'br', id: 1),
+      ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 2, country_code: 'br', id: 2)
+    ])
```

Comment:
About the rubocop alert. It expects your array to either be

1
```rb
assign(:item_seasons, [
                         ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 1, country_code: 'br', id: 1),
                         ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 2, country_code: 'br', id: 2)
                       ]
)
```

or 

2
```rb
assign(
  :item_seasons, 
  [
    ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 1, country_code: 'br', id: 1),
    ItemSeason.create!(produce_item_id: @produce_item.id, month_index: 2, country_code: 'br', id: 2)
  ]
)
```

I personally prefer 2, but that's up to you

Reply:
thanks! I kept reading and trying to rewrite it, but couldn't figure out what it was asking 😬 
I went with the second one 😄